raiden/blockchain_events_handler.py

```diff
@@ -519,9 +519,9 @@ def handle_channel_batch_unlock(raiden: "RaidenService", event: Event):
     )
     assert token_network_state is not None
```

Comment:
is the handle_channel_batch_unlock used both for light and full channels?

Reply:
Yes. I don't why all the functions here are both for light clients and full nodes